src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void printGreeting() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println("type in your command please");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printBye() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printEcho(String command) {
+        System.out.println("____________________________________________________________");
+        System.out.println(command.substring(5));
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printToDo(Task task, int size) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Got it. I've added this task: \n" + "\t" + task.toString());
+        System.out.println("Now you have " + size + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printDeadline(Task task, int size) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Got it. I've added this task: \n" + "\t" + task.toString());
+        System.out.println("Now you have " + size + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printEvent(Task task, int size) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Got it. I've added this task: \n" + "\t" + task.toString());
+        System.out.println("Now you have " + size + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
```

Comment:
A single general method to print all these message might be better for easier reading.